package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/pontem-network/wallet-adapter#readme",
   "scripts": {
-    "build": "rm -rf dist; tsc -p tsconfig.json"
+    "build": "rm -rf dist; tsc -p tsconfig.json; rm -rf node_modules"
```

Comment:
After removing node_modules we need to make yarn install each time when we want to build this package. It is a bad behavior. Better use .npmignore to ignore node_modules folder.

Reply:
I will fix it shortly